consensus/src/lib.rs

```diff
@@ -24,16 +19,16 @@ pub mod context;
 mod tests;
 pub mod transactions;
 
-pub use block::{
-    BlockVerifierService, PrePreparedBlock, VerifiedBlockInformation, VerifyBlockRequest,
-    VerifyBlockResponse,
-};
+pub use block::{BlockVerifierService, VerifyBlockRequest, VerifyBlockResponse};
 pub use context::{
     initialize_blockchain_context, BlockChainContext, BlockChainContextRequest,
     BlockChainContextResponse, BlockChainContextService, ContextConfig,
 };
 pub use transactions::{TxVerifierService, VerifyTxRequest, VerifyTxResponse};
 
+// re-export.
+pub use cuprate_types::service::{BCReadRequest, BCResponse};
```

Comment:
Maybe we should rename `service` -> `blockchain`?

Reply:
Will the database be the only service defined in `cuprate_types` ? Make sense to change it t `blockchain` since `cuprate-database` became `cuprate-blockchain`. In case we need to define types for other (future) services, we could consider each modules in cuprate_types to define one for a particular crate.

| module name | crate name |
| -------------------|--------------- |
| `cuprate_types::blockchain` | Define public types from `cuprate-blockchain` |
| `cuprate_types::rpc` | Define public types from `cuprate-rpc` |
| `cuprate_types::<crate>` | Define public types from `cuprate-<crate>` |

The current one make sense but the second require to agree on the crate layout imo, for the sake of maintainability. Maybe consider moving this table into `cuprate_types::crates::<crate>` instead.

Reply:
I don't think there will be any other services requests/ responses defined in `types`, maybe the txpool database